lib/spree_signifyd/create_signifyd_case.rb

```diff
@@ -6,6 +6,7 @@ def self.perform(order_number_or_id)
       Rails.logger.info "Processing Signifyd case creation event: #{order_number_or_id}"
       order = Spree::Order.find_by(number: order_number_or_id) || Spree::Order.find(order_number_or_id)
       order_data = JSON.parse(OrderSerializer.new(order).to_json)
+      Rails.logger.info "Order data sent to Signifyd for order #{order.number}: #{order_data}"
```

Comment:
Is `info` appropriate? This seems more like `debug` level info, as the output will be significant.